### PR TITLE
add google_qr_to_base64 method

### PR DESCRIPTION
**Pull request by harisadam**
more: https://github.com/jaredonline/google-authenticator/issues/53

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in google-authenticator.gemspec
 gemspec
+
+gem "sqlite3", "~> 1.3.6"
```
Comment (jaredonline) [inline Gemfile]: What's up with adding sqlite here?

**Comment by jaredonline** [comment]
Thanks for the PR @harisadam!

File: Gemfile (hunk 1 above)
Comment (harisadam) [reply Gemfile]: had some gem compatibility issue without this line, i can remove

**Comment by jaredonline** [comment]
@harisadam looks like tests are failing on:

> Please install the sqlite3 adapter: `gem install activerecord-sqlite3-adapter` (can't activate sqlite3 (~> 1.3.4), already activated sqlite3-1.4.0. Make sure all dependencies are added to Gemfile.) (RuntimeError)

**Comment by spejamchr** [comment]
I can get the tests to pass locally if I change https://github.com/jaredonline/google-authenticator/blob/fb2f9746ca033a2bc60874138058048a2d7bc449/google-authenticator.gemspec#L38

like so

```diff
-  gem.add_development_dependency "sqlite3"
+  gem.add_development_dependency "sqlite3",   "~> 1.3.6"
```

**Comment by jaredonline** [comment]
Hm, its odd that SQLite needs to be a dev dependency. I'll try to sort that out later and merge this now to unblock folks.

**Pull request merged**